showcase.json

```diff
@@ -4092,20 +3770,18 @@
     "logo": "/img/showcase/onicai_logo.png"
   },
   {
-    "id": "vfans",
-    "name": "VFans",
-    "oneLiner": "A decentralized content platform for publishing video/articles and subscribing through mobile payments.",
+    "id": "spotlabs",
```

Comment:
@radudiaconescu Can you please remove this?

Reply:
done